precompiles/warden/IWarden.sol

```diff
@@ -18,7 +18,7 @@ struct Key {
     uint64 id;
     uint64 spaceId;
     uint64 keychainId;
-    int32 keyType;
+    KeyType keyType;
```

Comment:
all these changes here looks unrelated to the title of this pr

Reply:
reverted